listenstore/listenstore/listenstore.py

```diff
@@ -258,6 +218,98 @@ def fetch_listens_for_range(self, uid, date_range, from_id, to_id, order, limit=
             else:
                 return
 
+class PostgresListenStore(ListenStore):
+    def __init__(self, conf):
+        ListenStore.__init__(self, conf)
+        self.postgre_connection = None
+        self.postgre_cursor = None
+        
+        self.postgres_conn = psycopg2.connect("dbname='' user='listenbrainz'")   
```

Comment:
Older configurations were hardcoded in listenstore which was not good. It was mentioned as TODO in code. :)
Added support for separate configuration, `listenstore.conf`, for ListenStore.
https://github.com/pinkeshbadjatiya/listenbrainz-server/blob/b1c922ab1c6ba7417160bec7411012df9769948c/listenstore/listenstore.conf.sample